beesblog.php

```diff
@@ -79,7 +79,7 @@ public function __construct()
     {
         $this->name = 'beesblog';
         $this->tab = 'front_office_features';
-        $this->version = '1.0.2';
+        $this->version = '2.0.0';
         $this->author = 'thirty bees';
```

Comment:
Version bumps happen when building a release, please.

Reply:
version number updated since theres new upgrade script for adding new columns. Leave the script name as upgrade-2.0.0.php and revert the module version number ?

Reply:
Yes, I'd do it this way.

Reply:
As you apparently have no Email and don't enjoy Slack either, please be aware of this: http://jump-ing.com/traumflugs/github_best_practices

There's also a module validation script, one would run it like `../../tools/validatemodule.sh` from within the module directory. This being not one of the standard modules, it probably doesn't validate at all, yet. Still things shouldn't get worse.

Reply:
yes, the workspace url please